graphql_compiler/tests/test_data_tools/data_tool.py

```diff
@@ -85,12 +85,13 @@ def generate_sql_integration_data(sql_test_backends):
         metadata,
         Column('animal_id', Integer, primary_key=True),
         Column('name', String(length=12), nullable=False),
+        Column('net_worth', Float, nullable=False),
```

Comment:
This doesn't look quite right. Isn't `net_worth` supposed to be a `Decimal`-typed field? I'm pretty sure most SQL databases support `Decimal` fields, though they might be fixed-precision in which case I'd say use 12 digits before the decimal point, and 2 decimal places.

Reply:
TIL, there's a `Numeric` that takes care of what you're describing.